cq_editor/widgets/occt_widget.py

```diff
@@ -143,14 +143,15 @@ def resizeEvent(self, event):
         self.view.MustBeResized()
     
     def _initialize(self):
-        
-        wins = {
-            'darwin' : self._get_window_osx,
-            'linux'  : self._get_window_linux,
-            'win32': self._get_window_win           
-        }
-        
-        self.view.SetWindow(wins[platform](self.winId()))
```

Comment:
Could you keep the original structure? This way it would be a one-line change
```python
self.view.SetWindow(wins.get(platform,self._get_window_linux)(self.winId()))
```

Reply:
The problem with that is that the platform reported by the current version of FreeBSD is freebsd12. When this inevitably changes in the not too distant future the code will no longer work. The changes I have made are in line with the recommendations of the python library reference which specifically says to use .startswith for this very reason.

Reply:
Does this work for you?

Reply:
Yes, that should work. To me the code looks a little fragile, but it should get the job done.

Reply:
Great.